scripts/update/Updater.php

```diff
@@ -1747,5 +1747,11 @@ public function update($initialVersion) {
             $this->setVersion('30.6.0');
         }
 
+        if ($this->isVersion('30.6.0')) {
+            AclProxy::applyRule(new AccessRule('grant', 'http://www.tao.lu/Ontologies/TAO.rdf#DeliveryRole', array('ext'=>'taoQtiTest', 'mod' => 'OfflineRunner')));
+
+            $this->setVersion('30.7.0');
```

Comment:
Shouldn't you register `OfflineQtiRunnerService` too?

Reply:
Fixed.